server/router/index.js

```diff
@@ -1,6 +1,6 @@
 const router = require('express').Router();
 const applicantRouter = require('./applicant-router');
 
-router.use(applicantRouter);
+router.use('/api/v1/applicant', applicantRouter);
```

Comment:
The `/api/v1` should be on the main router on app.js file and here you can put `/applicants`